### PR TITLE
Add EVs status into PKRS icon for routes and dungeons

### PR DESCRIPTION
**Pull request by Kasci**
If going for 100 completition, there is little to no information about, how much EVs are needed. Player needs to wait to see what pokemon are still being catched and check them one by one in Pokedex. Now it gives at least some overview, of how much is needed, it does not give it away, but also it gives valuable information.

### CODE REVIEW

File: src/components/dungeonView.html [hunk 1]
```diff
@@ -119,7 +119,8 @@ <h2 class="pageItemFooter" style="display: block; font-size: 1rem;">
 
                   <!--Pokérus image-->
                   <knockout data-bind="if: RouteHelper.minPokerusCheck(DungeonRunner.dungeon.allAvailablePokemon())">
-                      <img data-bind="attr:{ src: 'assets/images/breeding/pokerus/' + GameConstants.Pokerus[RouteHelper.minPokerus(DungeonRunner.dungeon.allAvailablePokemon())] + '.png'}"
+                      <img data-bind="attr:{ src: 'assets/images/breeding/pokerus/' + GameConstants.Pokerus[RouteHelper.minPokerus(DungeonRunner.dungeon.allAvailablePokemon())] + '.png',
+                            title: RouteHelper.dungeonPokerusEVs(DungeonRunner.dungeon)}"
```
Comment (Jaaslet) [inline src/components/dungeonView.html:123]: Use the tooltip attribute we are using other places

File: src/scripts/wildBattle/RouteHelper.ts [hunk 2]
```diff
@@ -38,6 +38,38 @@ class RouteHelper {
         return pokemonList;
     }
 
+    public static routePokerusEVs(route:number, region:GameConstants.Region): string {
+        const possiblePokemon: PokemonNameType[] = RouteHelper.getAvailablePokemonList(route, region);
+        if (this.minPokerus(possiblePokemon) == 3) {
```
Comment (Jaaslet) [inline src/scripts/wildBattle/RouteHelper.ts:43]: Use Gameconstant to make the code more readable

File: src/scripts/wildBattle/RouteHelper.ts [hunk 3]
```diff
@@ -38,6 +38,38 @@ class RouteHelper {
...
+            return 'All Pokémon on this route are resistant!';
+        }
+        const currentEVs = this.getEvs(possiblePokemon);
+        return 'EVs until all Pokémon are resistant on this route: ${currentEVs} / ${50 * possiblePokemon.length}.';
```
Comment (Jaaslet) [inline src/scripts/wildBattle/RouteHelper.ts:47]: I think the automatic eslint might have ruined the strings. It needs to be ` and not ', as far as i remember

**Review by Jaaslet — CHANGES_REQUESTED** [review]
I have not tested it, but i found some small things after looking through the code

File: src/scripts/wildBattle/RouteHelper.ts (hunk 3 above)
Comment (Kasci) [reply src/scripts/wildBattle/RouteHelper.ts]: Done :)

File: src/scripts/wildBattle/RouteHelper.ts (hunk 2 above)
Comment (Kasci) [reply src/scripts/wildBattle/RouteHelper.ts]: Done :)

File: src/components/dungeonView.html (hunk 1 above)
Comment (Kasci) [reply src/components/dungeonView.html:123]: Tooltip is possible only in Button, IMG uses title, same as images above (pokeballs)

**Review by Jaaslet — APPROVED** [review]
Looks good. Thanks for the PR

**Pull request merged**